addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -73,6 +73,14 @@ function factory(dependencies) {
         // Private
         //----------------------------------------------------------------------
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsActivityBoxVisible() {
+            return this.thread && this.hasActivityBox && this.threadActivities.length > 0;
```

Comment:
`threadActivities` is needed for the dependency, but it's best to directly access through the relation in code (`thread.activities`)